vtrace/platforms/linux.py

```diff
@@ -24,6 +24,8 @@
 
 from ctypes import *
 import ctypes.util as cutil
+from vtrace.platforms.posix import PT_CONTINUE
+from ptrace.binding.func import PTRACE_EVENT_EXEC, PTRACE_O_TRACEEXEC
```

Comment:
may have found a simpler way ( and clarity on what's going on ).  it looks like, if we ensure the ptrace(PT_TRACE_ME) gets executed, we no longer need to call ptrace(PT_ATTACH) from the parent, we can simply (as the parent) return from platformExec() and allow the regular old platformWait() code to catch the exec trap...  i'll work up a delta :)